frontend/src/css/GridHauptseite

```diff
@@ -0,0 +1,51 @@
+<div class="container">
```

Comment:
This file does not have an ending. And it is a mixture of html and css.

Reply:
deleted in next version